src/dataframe/dataframe.jl

```diff
@@ -453,7 +444,7 @@ end
 
 function insert_multiple_entries!(df::DataFrame,
                                   v::Any,
-                                  row_inds::AbstractVector{<:Integer},
+                                  row_inds,
```

Comment:
we can add it. I removed it because in general we have to also handle `Not` and `Colon`, but I have just checked that we will remove this method after deprecation anyway (new methods do not use it), so your proposal is OK.

Reply:
I prefer using explicit typing where possible, especially since this code is particularly hard to follow.

Reply:
Sure - I have changed this. The only problem is (but not in this case) that it is a big mental burden to write proper sets of explicit types as often using `Union` leads to method ambiguities due to Julia's dispatch rules (sorry for being lazy sometimes 😄).